index.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
This only affects this file right? Any chance it could change the behavior resulting in backward-incompatibilities?